doc/source/cluster/cloud.rst

```diff
@@ -123,7 +126,7 @@ Ray with cloud providers
 
     GCP Node Provider Maintainers (GitHub handles): wuisawesome, DmitriGekhtman, ijrsvt
 
-.. tabbed:: Aliyun
+.. tabbed:: Aliyun (community-maintained)
```

Comment:
Can you instead leave this as a note underneath? This increases the tab-width, and is also inconsistent with Azure.

Reply:
sgtm -- inconsistency unintentional

Reply:
Fixed.